package.json

```diff
@@ -1115,15 +1115,9 @@
           "default": "gpt-3.5-turbo",
           "enum": [
             "gpt-3.5-turbo",
-            "gpt-3.5-turbo-16k",
-            "gpt-3.5-turbo-0301",
-            "gpt-3.5-turbo-0613",
             "gpt-4",
-            "gpt-4-0314",
-            "gpt-4-0613",
-            "gpt-4-32k",
-            "gpt-4-32k-0314",
-            "gpt-4-32k-0613"
+            "gpt-4-turbo",
+            "gpt-4o"
```

Comment:
could you please keep the old values, but add the missing ones?

Reply:
ping @Illyism @kibertoad